### PR TITLE
feat: update & release system

### LINKED ISSUE #16: venum update
Update system / venom version

### PR DESCRIPTION
**Pull request by yesnault**
close #16

Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

File: venom.go [hunk 1]
```diff
@@ -2,8 +2,14 @@ package venom
 
 import (
 	"fmt"
+	"os"
 )
 
+// Version of Blinkstick
```
Comment (fsamin) [inline venom.go]: wrong copy & paste

File: venom.go [hunk 2]
```diff
@@ -2,8 +2,14 @@ package venom
...
+// One Line for this, used by release.sh script
+// Keep "const Version on one line"
+const Version = "0.9.0"
```
Comment (fsamin) [inline venom.go]: 0.1.0 ? or maybe it should be set on compilation

Comment (yesnault) [inline venom.go]: 0.9.0 -> first release will be "1.0.0", by executing ./release.sh 1.0.0

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+language: go
+go:
+- 1.8
```
Comment (fsamin) [inline .travis.yml]: you should set to `tip`

File: venom.go (hunk 2 above)
Comment (fsamin) [inline venom.go]: but why set 0.9 ? i would prefer "snapshot", if it's replaced on build time with tag name.

the very first public stable (1.0.0) will be released when we will considered it as stable - from a functionnal and design point of view (with plugin support for instance).

**Comment by fsamin** [comment]
my last comment isn't displayed as well
https://github.com/runabove/venom/pull/17#discussion_r102339954

**Comment by fsamin** [comment]
about https://github.com/runabove/venom/pull/17#pullrequestreview-23091926 ?

**Pull request merged**